documents/knowledge/elements.yaml

```diff
@@ -659,7 +659,7 @@
 - element: "?"
   name: Input
   arity: 0
-  description: Get the next input from the input source
+  description: Get and evaluate the next input (readline) from the input source.
```

Comment:
I think you should add that the global canvas and array can be affected by this.